core/src/blocks/curl.rs

```diff
@@ -154,8 +158,11 @@ impl Block for Curl {
         let response = request
             .execute_with_cache(env.project.clone(), env.store.clone(), use_cache)
             .await?;
-
-        Ok(json!(response))
+        headers_logs.extend(body_logs);
```

Comment:
let's do:
```
let all_logs = headers_logs;
all_logs.extend(body_logs);
```

And please add new lines surrounding pieces of code that are unrelated to each others :pray:

Reply:
@Uzay-G mind addressing this?